src/player.cpp

```diff
@@ -1989,13 +1986,16 @@ int player::run_cost( int base_cost, bool diag ) const
     }
     // Quad skates might be more stable than inlines,
     // but that also translates into a slower speed when on good surfaces.
-    if( is_wearing( "rollerskates" ) ) {
+    else if( worn_with_flag( "ROLLER_QUAD" ) ) {
         if( on_road ) {
             movecost *= 0.7f;
         } else {
             movecost *= 1.3f;
         }
     }
+    else if( is_wearing( "swim_fins" ) ) {
```

Comment:
All `else if` should be on one line with the closing brace, like this:
```
} else if( x ) {
```

I think visual studio tries to split them up, so you may want to disable automatic styling there. Or make it format it in project's style.

Reply:
I just added an else in front of the if but i'll go back and change the formating.  Also, I haven't gotten around to reinstalling VS since hard reseting my computer, I'm using VS Code right now.

Reply:
K done, little unsure about dealing with the comment though..

Reply:
Move comments to the case that actually covers them or get rid of them.
What you certainly don't want is this:
```
if( x ) {
    x-related-stuff();
    // y-related comment
} else if( y ) {
    y-related-stuff();
    // z-related comment
} else ...